webapp/src/components/news-list/NewsList.test.js

```diff
@@ -45,9 +45,8 @@ require('regenerator-runtime/runtime');
 
     const button = wrapper.find('button.sorter');
     button.trigger('click');
-
-    let firstDomElementInAscendingMode = wrapper.find('h1');
-    expect(firstDomElementInAscendingMode.text()).toContain("4");
+    expect(wrapper.vm.descending).toEqual(false);
+    expect(wrapper.vm.sortedItems[0]).toEqual({ title: "Title_2", votes: 4})
```

Comment:
Nooo, don't assert private properties of your component. Instead assert the rendered HTML when you use `mount` or the `props()` of the stubbed sub-components in case you use `shallowMount`.